apps/web/tests/setup/example.ts

```diff
@@ -64,7 +64,7 @@ class Example {
     }
 
     await expect(field.input).toHaveValue(value)
-    await expect(field.input).toHaveAttribute('type', type)
+    expect((await field.input.getAttribute('type')) ?? '').toBe(type)
```

Comment:
There's an update in playwright that makes `getAttribute('type')` return `null` instead of `''` so I had to add this change to make a bunch of tests pass after upgrading Playwright